src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -196,257 +196,257 @@ public sealed partial class ControlChars
-        public static void FileOpen(int FileNumber, string FileName, OpenMode Mode, OpenAccess Access = OpenAccess.Default, OpenShare Share = OpenShare.Default, int RecordLength = -1) { }
-        public static void FilePut(int FileNumber, bool Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, byte Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, char Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, decimal Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, double Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, float Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, int Value, long RecordNumber = -1) { }
-        public static void FilePut(int FileNumber, long Value, long RecordNumber = -1) { }
+        public static void FileOpen(int FileNumber, string FileName, Microsoft.VisualBasic.OpenMode Mode, Microsoft.VisualBasic.OpenAccess Access = Microsoft.VisualBasic.OpenAccess.Default, Microsoft.VisualBasic.OpenShare Share = Microsoft.VisualBasic.OpenShare.Default, int RecordLength = -1) { }
+        public static void FilePut(int FileNumber, System.Array Value, long RecordNumber = (long)-1, bool ArrayIsDynamic = false, bool StringIsFixedLength = false) { }
+        public static void FilePut(int FileNumber, bool Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, byte Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, char Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, System.DateTime Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, decimal Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, double Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, short Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, int Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, long Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, float Value, long RecordNumber = (long)-1) { }
+        public static void FilePut(int FileNumber, string Value, long RecordNumber = (long)-1, bool StringIsFixedLength = false) { }
+        public static void FilePut(int FileNumber, System.ValueType Value, long RecordNumber = (long)-1) { }
         [System.ObsoleteAttribute("This member has been deprecated. Please use FilePutObject to write Object types, or coerce FileNumber and RecordNumber to Integer for writing non-Object types. http://go.microsoft.com/fwlink/?linkid=14202")]
-        public static void FilePut(object FileNumber, object Value, object RecordNumber/* = -1*/) { }
```

Comment:
Note, we are losing this comment. Maybe it would be worth moving this to a .Manual file (possibly outside the PR).